weave-js/src/components/ToggleButtonGroup/types.tsx

```diff
@@ -0,0 +1,5 @@
+export enum ToggleButtonGroupSizes {
+  small = 'small',
+  medium = 'medium',
+  large = 'large',
```

Comment:
super nit - i think this is the convention
```
export enum ToggleButtonGroupSizes {
  Small,
  Medium,
  Large,
}
```
I don't think you have a need for them to be backed by strings here